subpackages/property/source/unit_threaded/property.d

```diff
@@ -34,19 +34,10 @@ void check(alias F, int numFuncCalls = 100)
     static assert(is(ReturnType!F == bool),
                   text("check only accepts functions that return bool, not ", ReturnType!F.stringof));
 
-    scope random = Random(seed);
-
-    // See https://github.com/atilaneves/unit-threaded/issues/187 for why
-    auto createGenerator(ref Random random) {
-        return RndValueGen!(Parameters!F)(&random);
-    }
-
-    // It might be that some projects don't use dip1000 and so
-    // createGenerator isn't safe
-    static if(isSafe!createGenerator)
-        scope gen = createGenerator(random);
-    else
-        scope gen = () @trusted { return createGenerator(random); }();
+    // `Random` could be put on the stack, but only with -dip1000:
```

Comment:
Won't this then break unless DIP1000 is used?

Reply:
I now changed it to allocate on the heap, since it doesn't look like a performance critical function. The comment describes a possible enhancement that is currently blocked by bugs.